pkg/chains/signing.go

```diff
@@ -141,90 +141,93 @@ func (ts *TaskRunSigner) SignTaskRun(ctx context.Context, tr *v1beta1.TaskRun) e
 	var merr *multierror.Error
 	extraAnnotations := map[string]string{}
 	for _, signableType := range enabledSignableTypes {
+		if signableType.Enabled(cfg) {
```

Comment:
let's check the opposite to make this more readable:

```go
if !signableType.Enabled(cfg) {
   continue
}
```

Reply:
changed in `signing.go` and `verifier.go`